hawc/apps/summary/templates/summary/datapivot_type_selector.html

```diff
@@ -12,7 +12,9 @@ <h1>Create data pivot</h1>
   <hr>
 
   <p class="lead">Option #2 (Input-file)</p>
-  <p class='form-text text-muted'>Create a data-pivot based on an uploaded Excel file of data. Your settings will always be applied to the uploaded dataset; therefore the visualization will be constant, but will not use the latest information from the HAWC database.</p>
+  <p class='form-text text-muted'>
+    Create a data-pivot based on an uploaded Excel file of data. Your settings will always be applied to the uploaded dataset; therefore the visualization will be constant, but will not use the latest information from the HAWC database. However, if you <a href="{% url 'summary:dataset_interactivity' %}" onclick="return window.app.HAWCUtils.newWindowPopupLink(this);">design your dataset</a> appropriately, HAWC interactivity is available.
```

Comment:
revised language here instead of the `click here` which was originally proposed; generally style guides recommend working hyperlinks into a sentence for readability